job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/ExecuteDslScripts.java

```diff
@@ -302,7 +313,7 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
                     dslScriptLoader = new JenkinsDslScriptLoader(jobManagement);
                 }
 
-                GeneratedItems generatedItems = dslScriptLoader.runScripts(scriptRequests);
+                GeneratedItems generatedItems = dslScriptLoader.runScripts(scriptRequests, additionalParameters);
```

Comment:
@daspilker Sure, I can do that.
I wanted to keep the distinction in the source code between environment variables and the additional params. What about putting them all into the environment map under the key "additionalParameters" or something to mirror how they're invoked from the job DSL?

Also, what about conflicts between actual environment variables and additional parameters. I don't like the idea of mixing the two without clear semantics.